modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -267,11 +306,28 @@ const Combobox = ({
     focusInput();
   };
 
+  const getGroupIndex = (itemIndex: number | null) => {
+    if (
+      itemIndex != null &&
+      autocompleteItems &&
+      autocompleteItems.length &&
```

Comment:
maybe to shorten if statement? `autocompleteItems?.length`

Reply:
Last I checked null coalescing and optional chaining wasn't available in CKR, will double check again.

Reply:
It will be in 4.0! Not in master yet